virtool/samples/api.py

```diff
@@ -180,21 +180,16 @@ async def get(req):
 
     document["caches"] = caches
 
-    if document["ready"] is True:
-        # Only update file fields if sample creation is complete.
-        for index, file in enumerate(document["files"]):
-            snake_case = document["name"].replace(" ", "_")
-
-            file.update({
-                "name": file["name"].replace("reads_", f"{snake_case}_"),
-                "download_url": file["download_url"].replace("reads_", f"{snake_case}_"),
-                "replace_url": f"/upload/samples/{sample_id}/files/{index + 1}"
-            })
-
     document = await virtool.subtractions.db.attach_subtractions(db, document)
     document = await virtool.samples.db.attach_labels(pg, document)
     document = await virtool.samples.db.attach_artifacts_and_reads(pg, document)
 
+    if document["ready"]:
+        for index, file in enumerate(document["reads"]):
```

Comment:
```suggestion
        for file in document["reads"]:
```

Reply:
Good catch. And I did not know you could suggest changes...